spec/functions/datetime.md

```diff
@@ -282,11 +282,14 @@ the functions `:datetime`, `:date`, and `:time`.
   - `local`
   - `UTC`
 
-> [!NOTE]
-> The value `local` permits a _message_ to convert a date/time value
-> into a [floating](https://www.w3.org/TR/timezone/#floating) time value
-> (sometimes called a _plain_ or _local_ time value) by removing
-> the association with a specific time zone.
+The value `local` corresponds to the default time zone.
```

Comment:
The value `local` is meant to remove the time zone (it floats the value). This is different from the default time zone. Consider:

```
.local $event = {|2025-06-28T19:00:00Z| :datetime}
{{The Annual Barn Dance will be on {$event :date timeZone=local}.}}
```

The above formats as "June 28, 2025" regardless of where the message is viewed, vs. potentially formatting as "June 29, 2025" or "June 27, 2025" depending on the default time zone. The name `local` I find misleading. The Temporal synonym is `plain`, which I also find unhelpful.

Reply:
Oh, so `local` is effectively "local" to the input value, rather than being "local" to the time zone of the environment where the formatting is being done, yes? I had understood it the other way around.

We should probably use a different term that is more implicitly obvious about its meaning. Maybe `input`?

Reply:
As expressed elsewhere, I hate the term `local` for this. It makes more sense in HTML forms, where presumably the user in inputting local wall time, but no sense in a context like MF. Temporal uses the term `plain` for this